### ISSUE TITLE
EmojiToolbarPopover error: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by Nightbr** [Foundry376/Mailspring#1189]
Hello guys,

##### What operating system are you using?

Linux Mint 19 Cinnamon

##### What version of Mailspring are you using?
1.5.1-da141eaf
--

**Bug?**

When I click on EmojiToolbar when writing email, it does nothing. I dig it deeper and there is an error:

![image](https://user-images.githubusercontent.com/4228646/47637487-2213d780-db5c-11e8-8183-6f08a7323c7d.png)

```
TypeError: Cannot read property 'length' of undefined
    at EmojiToolbarPopover.getStateFromStore (/src/components/composer-editor/emoji-toolbar-popover.jsx:135:59)
    at new EmojiToolbarPopover (/src/components/composer-editor/emoji-toolbar-popover.jsx:13:73)
    at constructClassInstance (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:109:354)
    at beginWork (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:133:440)
    at d (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:161:93)
    at f (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:161:417)
    at g (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:162:149)
    at m (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:169:173)
    at w (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:168:413)
    at batchedUpdates (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:171:357)
    at tc (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:54:269)
    at vd (/usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:71:249)
h @ /usr/share/mailspring/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:164
```
Let me know if you need more logs or debug!

Thanks!

**Comment by bengotow** [comment]
Oh wow thank you for reporting this! It looks like we're localizing the section names in the wrong place , so this code only works if you run Mailspring in English. Will get this fixed in the next release!

**Comment by Nightbr** [comment]
This is fix in 1.5.2, I just update and it works! Thanks!
